kie-bom/pom.xml

```
@@ -2680,7 +2680,7 @@
         <artifactId>business-central-webapp</artifactId>
         <version>${version.org.kie}</version>
         <type>jar</type>
-        <classifier>thorntail</classifier>
+        <classifier>bootable</classifier>
```

could we take standalone classifier ?
```suggestion
        <classifier>standalone</classifier>
```

---

Yes we can.